nginx/prod.nginx.conf

```diff
@@ -1,11 +1,24 @@
 limit_req_zone $binary_remote_addr zone=api_limit:10m rate=25r/s;
-limit_req_zone $binary_remote_addr zone=nginx_limit:10m rate=30r/s;
+limit_req_zone $binary_remote_addr zone=nginx_limit:10m rate=40r/s;
 
 upstream api {
   least_conn;
   server api:8081 weight=10 max_fails=3 fail_timeout=30s;
 }
 
+proxy_cache_path /nginx-cache levels=1:2 keys_zone=my_cache:30m inactive=60m use_temp_path=off max_size=3g;
+
+map $sent_http_content_type $expires {
+    default                 off;
+    text/html               7d;
+    text/css                7d;
+    text/js                 7d;
+    text/javascript         7d;
+    application/javascript  7d;
+    application/json        7d;
+    ~image/                 max;
```

Comment:
How come we don't have max for everything?

Reply:
If we deploy a new release, the browser should request new resources. I think `js` and `css` have a hash in the name, so we could have `max` for these. But for `json`, I wouldn't cache it forever.

Reply:
When we deploy a new release doesn't everything get reset?

Reply:
Yes, but if the URL to the js or css would stay the same, the browser might think it's the same (cached) resource. I'm not sure if  `Etag`s help with this. Anyway I've increased all non-json resources to `max`.